tube-player/README.md

```diff
@@ -37,29 +40,52 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
-      - NoneTemplate
-      - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+### Environment setup & Inner loop
+
+- [Hot reload](xref:Uno.Features.HotReload)
+- [Uno.Check](xref:UnoCheck.UsingUnoCheck)
+
+### Cross-platform development
+
+- [Resizetizer](xref:Uno.Resizetizer.GettingStarted)
+  - [UnoIcon](xref:Uno.Resizetizer.GettingStarted#unoicon)
+  - [UnoSplashScreen](xref:Uno.Resizetizer.GettingStarted#unosplashscreen)
+- [Template Wizard](xref:Uno.GettingStarted.UsingWizard)
+
+### Battle-tested features for apps
+
+- [C# Markup](xref:Uno.Extensions.Markup.Overview)
+- [Hosting environments](xref:Uno.Extensions.Hosting.Overview#hosting-environments)
+- [Navigation with regions](xref:Uno.Extensions.Navigation.Advanced.Panel)
+- [Refit endpoints](xref:Uno.Extensions.Http.HowToRefit)
+- [Serialization](xref:Uno.Extensions.Serialization.Overview)
+
+### Clean architecture, simplified
+
+- [MVUX](xref:Uno.Extensions.Mvux.Overview)
+  - Creating [Feeds](xref:Uno.Extensions.Mvux.Feeds)
+  - Consuming feed data using [FeedView](xref:Uno.Extensions.Mvux.FeedView) with a custom [ErrorTemplate](xref:Uno.Extensions.Mvux.FeedView#errortemplate) and [NoneTemplate](xref:Uno.Extensions.Mvux.FeedView#nonetemplate)
+  - [Paginating](xref:Uno.Extensions.Mvux.Advanced.Pagination) feed data
+
+### Figma
+
 - [Figma plugin](https://aka.platform.uno/uno-figma)
-  - C# Markup
-  - Color overrides
-- [Color overrides](xref:uno.themes.material.getstarted)
-- [DSP import](xref:Uno.Material.DSP)
-- [Material design theme](xref:uno.themes.material.getstarted)
-- [Toolkit](xref:Toolkit.GettingStarted)
-  - [CardContentControl](xref:Toolkit.Controls.Card)
+- [Design to code](xref:Uno.Figma.GetStarted.DesignToCode)
+- [Custom colors](xref:Uno.Figma.Learn.Developers.CustomColors)
+- [Export as C# Markup](xref:Uno.Figma.Learn.Developers.Tabs.Export#c-markup-export)
+
+### Theming
+
+- [Material design theme](xref:Uno.Themes.Material.GetStarted)
+- [Color overrides](xref:Uno.Themes.Material.Colors)
+- [DSP tooling](xref:Uno.Themes.Material.DSP)
+
+### Efficient and reusable UI components
+
+  - [AutoLayout](xref:Toolkit.Controls.AutoLayoutControl)
+  - [CardContentControl](xref:Toolkit.Controls.Card#cardcontentcontrol)
+  - [ExtendedSplashScreen](xref:Toolkit.Controls.ExtendedSplashScreen)
   - [NavigationBar](xref:Toolkit.Controls.NavigationBar)
```

Comment:
@kazo0, @lukeblevins 
I think having Toolkit overview on top of this list could be useful here (similar to MVUX overview above)
What do you think ?